GameData/Skopos/Contracts/ExperimentalComsats/intermittent_paris_moscow_tv.cfg

```diff
@@ -12,9 +12,9 @@ CONTRACT_TYPE {
   sortKey = 050
   title = Experimental Paris-Moscow Television
   // Mostly pilfered from NASA TM X-70435 p. 11-1.
-  description = <b>Program: Experimental Telecommunications Satellites (Skopos Beta)<br>Type: <color=green>Required</color></b><br><br>As part of a Franco-Soviet agreement on colour television, we are considering using the space segement of the Soviet television network to relay television between Paris and Moscow.
+  description = <b>Program: Experimental Telecommunications Satellites (Skopos Beta)<br>Type: <color=green>Required</color></b><br><br>As part of a Franco-Soviet agreement on color television, we are considering using the space segment of the Soviet television network to relay television between Paris and Moscow.
```

Comment:
Let’s stick to en-GB-oxendict orthography, which is what I have used throughout; this change misses many british spellings (of colour and other words) and makes the result less consistent.

Reply:
Okay, fixed